lib/reverse_characters.rb

```diff
@@ -1,2 +1,21 @@
+#To test my code in repl.it, created an str variable using the line below:
+#str = "Today is a good day".split("")
+
 def reverse_characters(str)
+arr_end = str.length - 1
```

Comment:
Indentation is an issue here. Lines 5-20 should all be indented one space to be contained within the method block (which begins on line 4 and ends on line 21).